src/Console/Shell.php

```diff
@@ -242,12 +249,14 @@ public function startup()
      */
     protected function _welcome()
     {
+        $phpInfo = explode('~', phpversion());
+        $this->phpVersion = (isset($phpInfo[0]) ? $phpInfo[0] : phpversion());
```

Comment:
I would return the full php version as `phpversion()` returns it anyway. And I do not see the reason to use a variable then.